app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<h1> Welcome To Crime Time! </h1>
+<section>
+  <%= link_to "Login with Google"  %>
+</section>
+
+<section class='national-stats'>
+  <h3> National Stats: </h3>
+  <p class='aggrivated-assault'> </p>
+  <p class='homicide'></p>
+  <p class='rape'></p>
+  <p class='property-crime'></p>
+  <p class='arson'></p>
+</section>
+
+<section class='top-ten'>
+  <h3> Top Ten Most Wanted: </h3>
+  <p class='fugitive-one'> </p>
+  <p class='fugitive-two'></p>
+  <p class='fugitive-three'></p>
+  <p class='fugitive-four'></p>
+  <p class='fugitive-five'></p>
+  <p class='fugitive-six'></p>
+  <p class='fugitive-seven'></p>
+  <p class='fugitive-eight'></p>
+  <p class='fugitive-nine'></p>
+  <p class='fugitive-ten'></p>
```

Comment:
We need to discuss how to  package and receive theses pieces of info from the controller